packages/theme/src/components/tooltip.ts

```diff
@@ -1,58 +1,22 @@
-import { cva, VariantProps } from 'class-variance-authority'
 import clsx from 'clsx'
 
-const tooltipCva = cva(['rounded-lg px-3 py-2 text-xs font-semibold'], {
-  variants: {
-    theme: {
-      light: ['bg-white', 'text-neutral-800'],
-      dark: ['bg-neutral-800', 'text-white'],
-    },
-  },
-  defaultVariants: {
-    theme: 'light',
-  },
-})
+const tooltipClsx = ({ className = '' }: { className?: string }) =>
+  clsx(
+    'rounded-lg',
+    'px-3',
+    'py-2',
+    'text-xs',
+    'font-semibold',
+    'bg-background-tooltip',
+    'text-text-primary',
+    className,
+  )
 
-const tooltipArrowCva = cva([], {
-  variants: {
-    theme: {
-      light: ['fill-white'],
-      dark: ['fill-neutral-800'],
-    },
-  },
-  defaultVariants: {
-    theme: 'light',
-  },
-})
+const tooltipArrowClxs = 'fill-background-tooltip'
```

Comment:
Use `fill-{semantic}-{number}` ex: fill-neutral-600 is okay.
But using `fill-{semantic}-{variant}-{...}` (ex: fill-primary-solid, fill-background-tooltip)  not work here.